schema/lc.proto

```diff
@@ -68,6 +71,8 @@ message ZItemExt {
   google.protobuf.Timestamp timestamp = 2;
   google.protobuf.Timestamp apikeyRevoked = 3;
   string ledgerName = 4;
+  string publicKey = 5;
```

Comment:
do we use it anywhere in vcn so it needed to be updated ? could not find ZScanExt ?

Reply:
No it is not used anymore. I added it only for consistency. We will clean soon proto schema from legacy objects I suppose